src/Masa.Utils.Exceptions/MasaExceptionContext.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+namespace System;
+
+public class MasaExceptionContext
+{
+    public Exception Exception { get; set; }
+
+    public HttpContext HttpContext { get; }
+
+    public bool ExceptionHandled { get; set; }
+
+    public int StatusCode { get; set; }
+
+    public string? Message { get; set; }
+
+    public string ContentType { get; set; }
```

Comment:
Maybe we can provide `ErrorCode` property, the type is `string?`. It can be use for I18n message.